### DISCUSSION TITLE
Sandy Bridge etc builds

### DISCUSSION THREAD

**Discussion by tormento** [DJATOM/x264-aMod#2] [General]
Do you mind creating various CPU builds as you do with x265?

I have a old but still good Sandy Bridge and I'd like to squeeze it for a few more months.

Thanks!

**Comment by DJATOM** [comment]
Sure, I'll try to make them tomorrow.

**Reply by DJATOM** [reply]
Done.

**Reply by tormento** [reply]
Where?

**Reply by DJATOM** [reply]
Old archive was replaced with new one. Just re-download, it's all inside.

**Reply by tormento** [reply]
Ah ok, will the next releases include them as default?

**Reply by DJATOM** [reply]
Yes, I made a script for compiling them sequentially.
